tests/unit/TableTest.php

```diff
@@ -327,4 +273,28 @@ public function testDeleteColumn()
 
         $this->assertNull($r);
     }
+
+    protected function createTestTable(string $tableName,bool $hasAge = true):Morebec\YDB\Table
```

Comment:
this hasAge parameter is not really scalable. If we ever need to create more tests that have different additional fields,
that would mean adding a parameter per addition fields.

what we could do instead could be this:

```php
public function createTestTable(string $tableName, array $additionalColumns = [])
```
That way, tests that need additional columns could simply use this parameter instead.